src/bench/base58.cpp

```diff
@@ -13,28 +13,30 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
-        17, 79, 8, 99, 150, 189, 208, 162, 22, 23, 203, 163, 36, 58, 147,
-        227, 139, 2, 215, 100, 91, 38, 11, 141, 253, 40, 117, 21, 16, 90,
-        200, 24
+    std::array<unsigned char, 32> buff = {
```

Comment:
tiny nit: I might prefer this if it kept the benchmarked-against data on stack instead of heap, but it doesnt matter much.